### ISSUE TITLE
Doesn't work in offline mode

### ISSUE THREAD

**Issue by NTCoding** [daattali/gslides-betternotes-extension#9]
This is a great extension - the image of the slide is actually useful now :laughing: 

Unfortunately, it doesn't work in offline mode, which is a massive shame. I don't know if it's possible to do that, but it would be very useful.

Regardles, thank you very much for making and sharing this incredibly useful tool.

**Comment by daattali** [comment]
Can you clarify what it means to be in offline mode?  Do you mean you open a presentation and then disconnect from the internet, or is there some special "offline mode" that you can somehow activate for a document?  And do you know if other extensions work for such "offline mode"? (I'm not familiar with offline at all :smile: )

**Comment by NTCoding** [comment]
Hey,

To get into offline mode, firstly I use the google drive desktop app for offline sync. I also have the google drive chrome extension installed. I can then open up and use google slides files (.gslides) stored on my machine even when I don't have an internet connection. I do this as a backup so when I talk at conferences I am not dependent on a network connection.

Cheers

**Comment by daattali** [comment]
Thanks.  Are you using any other simple extensions that are meant to work
in normal mode and you've tried in offline? I'm wondering if it's expected
for extensions to work in offline mode or not, my intuitive guess is that
it's not :)

**Comment by NTCoding** [comment]
I've just tested a couple and they work. Lastpass, emojistuff, and a google translate (fails when it tries to hit the remote server of course). I had a quick search and found this http://stackoverflow.com/questions/16490752/why-is-this-extension-not-working-offline-code-included but I don't really understand chrome extensions so don't know if it's useful or not.

**Comment by NTCoding** [comment]
The official docs suggest it should just work in offline mode https://developer.chrome.com/apps/offline_apps#manifest so perhaps it's related to google slides.

**Comment by daattali** [comment]
Thanks for all the details. I was able to reproduce after enabling offline mode and everything. Unfortunately it looks like the extension isn't even called by Chrome when I open the speaker notes in offline mode. Maybe while extensions generally work offline (I tested my other extensions and they do work offline), Google Drive Offline does something special and because doesn't go through the regular extension system. I'm not sure...
